main_pullrequest.go

```diff
@@ -42,9 +42,14 @@ func processGitHubPullRequest(
 	log.Debugf("Processing pull request action %s", action)
 	switch action {
 	case "opened", "edited", "reopened", "synchronize", "ready_for_review":
+		msgDetails := "see <a href=\"https://console.cloud.google.com/kubernetes/" +
+			"deployment/us-east1/company-websites/default/test-runner-mender-io/logs?" +
+			"project=gp-kubernetes-269000\">logs<\a> for details."
 		// We always create a pr_* branch
 		if prRef, err = syncPullRequestBranch(log, pr, conf); err != nil {
 			log.Errorf("Could not create PR branch: %s", err.Error())
+			msg := "Kitty, there was an error syncing branches, " + msgDetails
```

Comment:
Kitty? Did you mean `:smile_cat:`?

Reply:
In any case I don't think we need it. Instead I would add a danger or problem kind of emoji.

Reply:
I was just following the ticket's description, IMO both works fine, but if you think removing it is better, then let's change that! 🙃

Reply:
Haha, I didn't expect anyone writing such a thing in the acceptance criteria... :sweat_smile: